pytket_qir/converter.py

```diff
@@ -103,12 +103,13 @@ def __init__(
         wasm_int_size: int = 32,
         qir_int_size: int = 64,
         circuit: Optional[Circuit] = None,
-        module: Optional[SimpleModule] = None,
+        module: Optional[Module] = None,
```

Comment:
Is this just a rename?

Reply:
No, this is my custom type to extend `pyqir` module functionality to arbitrary gateset.